README.md

```diff
@@ -63,3 +69,46 @@ You can fork this repo to create your own boilerplate, and use it with `vue-cli`
 ``` bash
 vue init username/repo my-project
 ```
+
+## Configuring your Web server to prevent caching
+
+When you run the production build and deploy the application to a web server, the browser will call the service worker on the first visit and will download and cache all resources. 
+When the browser accesses the site a second time, it will use the cached resources.
+To notice changes in your app it is crucial that the browser downloads the latest version `service-worker.js` from your server.
+The browser will only do this when the HTTP Header of the service worker indicates that a previous download has expired.
+If a new service worker is available, it will download the new contents in the background.
+Once the download is complete, the default behavior is that the new content will be shown on the next visit of the site. 
+You can change the template's `service-worker-prod.js` to display the message to the user to reload the page, or issue a `window.location.reload()` automatically to trigger an update depending on your application's needs.
+
+Depending on your setup, you might want to have a short caching time for your service worker or no caching at all.
+At minimum, you should add caching headers for `service-worker.js` and `index.html`.
```

Comment:
Can you rephrase "you should add caching headers" to make it clear that the headers you add should disable caching?